flocker/control/test/test_persistence.py

```diff
@@ -38,17 +38,17 @@
     Port, Link, Leases, Lease, BlockDeviceOwnership, PersistentState,
     )
 
-# The UUID values for the Dataset and Node in TEST_DEPLOYMENT match
-# those in the versioned JSON configuration files used by tests in this
-# module. If these values are changed, you will also need to regenerate
-# the test JSON files using the scripts provided in the
-# flocker/control/test/configurations/ directory, using the correct
-# commit checkout to generate JSON appropriate to each config version.
+# The UUID values for the Dataset and Node in the following TEST_DEPLOYMENTs
+# match those in the versioned JSON configuration files used by tests in this
+# module.  If these values are changed, you will also need to regenerate the
+# test JSON files using the scripts provided in the
+# flocker/control/test/configurations/ directory, using the correct commit
+# checkout to generate JSON appropriate to each config version.
```

Comment:
:question: Yeah, so the docs say nothing about whether the script prints or writes the output to a file.